File: app.client/src/api/models/BankTransactionListVM.ts
export type BankTransactionListVM = {
    transactionId?: string;
    bankId?: string;
    bankName?: string;
    approvalId?: string;
    approvalName?: string;
    transactionType?: string;
    amount: number;
    deposit: number;
    withdrawal: number;
    runningBalance: number;
    createdDate?: string;
    createdBy?: string;
};

File: App.Application/Features/Global/BankTransactions/Queries/GetBankTransactionsList/BankTransactionListVM.cs
using System;

namespace OOH.Application.Features.Global.BankTransactions.Queries.GetBankTransactionsList
{
    public class BankTransactionListVM
    {
        public string? TransactionId { get; set; }
        public string? BankId { get; set; }
        public string? VendorId { get; set; }
        public string? BankName { get; set; }
        public string? ApprovalId { get; set; }
        public string? ApprovalName { get; set; }
        public string? TransactionType { get; set; }
        public decimal Amount { get; set; }
        public decimal Deposit { get; set; }
        public decimal Withdrawal { get; set; }
        public decimal RunningBalance { get; set; }
        public string? CreatedDate { get; set; }
        public string? CreatedBy { get; set; }
    }
}
